src/library/scala/concurrent/Future.scala

```diff
@@ -566,18 +567,29 @@ object Future {
    */
   object never extends Future[Nothing] {
 
-    private[this] final val notGoingToHappen = new CountDownLatch(1)
-
     @throws[TimeoutException]
     @throws[InterruptedException]
     override final def ready(atMost: Duration)(implicit permit: CanAwait): this.type = {
       import Duration.{Undefined, Inf, MinusInf}
       atMost match {
         case u if u eq Undefined => throw new IllegalArgumentException("cannot wait for Undefined period")
-        case `Inf`               => notGoingToHappen.await()
+        case `Inf`               =>
```

Comment:
Actually, what's the point of using backticks here?

Reply:
[Future-proofing](https://contributors.scala-lang.org/t/scala-3-should-variables-name-format-have-same-semantics/3765), oh and also, that's a future pun.

Reply:
That reminds me, I have to backtick my patterns and also get my earthquake kit ready.

Reply:
You can `never` be too careful in this `case`.

Reply:
"You're only saying `never` because no one ever has." - Princess Bride